runners/samza/src/main/java/org/apache/beam/runners/samza/translation/ReshuffleTranslator.java

```diff
@@ -42,6 +42,16 @@
 public class ReshuffleTranslator<K, InT, OutT>
     implements TransformTranslator<PTransform<PCollection<KV<K, InT>>, PCollection<KV<K, OutT>>>> {
 
+  private final String prefix;
+
+  ReshuffleTranslator(String prefix) {
+    this.prefix = prefix;
+  }
+
+  ReshuffleTranslator() {
+    this("rhfl-");
```

Comment:
Was the dropping of the s in "rshfl" intentional?

Reply:
fixed